src/azure-cli/azure/cli/command_modules/monitor/commands.py

```diff
@@ -244,3 +250,6 @@ def load_command_table(self, _):
         g.command('list', 'list_intelligence_packs')
         g.command('enable', 'enable_intelligence_pack')
         g.command('disable', 'disable_intelligence_pack')
+
+    with self.command_group('monitor', metric_alert_sdk, custom_command_type=monitor_general_custom) as g:
+        g.custom_command('clone', 'clone_existed_settings')
```

Comment:
You only clone metric alerts in _clone_monitor_metrics_alerts, will you clone all settings in this command or you only clone alerts?

Reply:
yeah. good question. The original plan is to clone all settings. After talking with Arun, he said that currently the feature request is to clone metric alerts only. But I want to leave enough space for future change. So I don't change the command name.

Reply:
what if later we need to support specific other settings to clone? should we add a parameter like --settings metric-alert

Reply:
good suggestion. I think we can do that. Currently, we can make `--settings` have default value `metric-alert`